src/clr-addons/brand-avatar/brand-avatar.ts

```diff
@@ -25,16 +48,31 @@ export class ClrBrandAvatar {
   @HostBinding('class.clr-brand-avatar') avatar = true;
 
   @Input('clrBrand') brand: string;
+  @Input('clrSize') size = 24;
 
   @ViewChild('avatar', { static: true }) avatarElement: ElementRef;
 
   static DEFAULT_ICON_SHAPE = 'new-car-private';
 
+  defaultIcon = false;
+
+  constructor(private renderer: Renderer2) {}
+
+  ngAfterViewInit(): void {
+    this.renderer.setStyle(this.avatarElement.nativeElement, 'width', this.size + 'px');
+    this.renderer.setStyle(this.avatarElement.nativeElement, 'height', this.size + 'px');
+  }
+
+  ngAfterContentInit(): void {
```

Comment:
i guess this should be done more frequent, either in a getDefaultIcon method or when "clrBrand" has been set. I refer to async loading of data when the brand comes in at any time